variables.tf

```diff
@@ -77,12 +240,31 @@ variable "public_access_block" {
   }
 }
 
+variable "server_side_encryption" {
```

Comment:
As mentioned on the resource, I think this variable can be removed

Reply:
Removed